catalog/confluence/catalog-confluence-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,108 @@
+            type="Long"
+            default="60000"
+            description="Availability polling interval in milliseconds"/>
+    </OCD>
+
+
+    <OCD description="DDF Confluence Connected Source" name="Confluence Connected Source"
+         id="Confluence_Connected_Source">
+
+        <AD name="Confluence Rest URL" id="endpointUrl" required="true"
+            type="String"
+            description="The Confluence Rest API endpoint URL. Example: https://<host>:<port>/rest/api/content"/>
+
+        <AD name="Username" id="username" required="false"
+            type="String"
+            default=""
+            description="Username to use with HTTP Basic Authentication. This auth info will overwrite any federated auth info. Only set this if the Confluence endpoint requires basic authentication."/>
+
+        <AD name="Password" id="password" required="false"
+            type="Password"
+            default=""
+            description="Password to use with HTTP Basic Authentication. This auth info will overwrite any federated auth info. Only set this if the Confluence endpoint requires basic authentication."/>
+        <AD name="Include Page Contents In Results" id="includePageContent" required="false"
+            type="Boolean"
+            default="false"
+            description="Flag indicating if Confluence page contents should be included in the returned results"/>
+        <AD name="Include Archived Spaces" id="includeArchivedSpaces" required="false"
+            type="Boolean"
+            default="false"
+            description="Flag indicating if archived confluence spaces should be included in search results"/>
+        <AD name="Exclude Confluence Spaces" id="excludeSpaces" required="false"
+            type="Boolean"
+            default="false"
+            description="Flag indicating if the list of Confluence Spaces should be excluded from searches instead of included"/>
+        <AD name="Confluence Spaces" id="confluenceSpaces" required="false"
+            type="String" cardinality="1000"
+            description="The confluence spaces to include/exclude from searches. If no spaces are specified all visible spaces will be searched"/>
+        <AD name="Additional Attributes" id="additionalAttributes" required="false"
+            type="String" cardinality="100"
+            description="Additional attributes to add to confluence metacards returned from this source."/>
+        <AD name="Availability Poll Interval" id="availabilityPollInterval" required="false"
+            type="Long"
+            default="60000"
+            description="Availability polling interval in milliseconds"/>
+    </OCD>
+
+    <Designate pid="Confluence_Federated_Source" factoryPid="Confluence_Federated_Source">
+        <Object ocdref="Confluence_Federated_Source"/>
+    </Designate>
+
+    <Designate pid="Confluence_Connected_Source" factoryPid="Confluence_Connected_Source">
```

Comment:
Looks like all the other DDF sources use the factory pid for the pid in their metatypes which is what I was basing this one off of. Let me know if you still want this changed.